### ISSUE TITLE
Implement "blank transition" VM procedure

### ISSUE THREAD

**Issue by dr-orlovsky** [LNP-BP/rgb-node#67]
When some UTXO has an assigned state under multiple contracts with multiple schemas it is required that when one of the stage is transitioned and UTXO is spent the rest of states under all other contracts will be assigned to new outputs. For this, VM should support "blank transition" procedure, which do not modifies state and just re-assignes it to some other output.

https://github.com/LNP-BP/rgb-node/blob/4e9e632c85ae808e81044b4147076879f9701ace/src/contracts/fungible/runtime.rs#L248

**Comment by rajarshimaitra** [comment]
I think this is solved by https://github.com/rgb-org/rgb-node/pull/148

**Comment by dr-orlovsky** [comment]
No, they are unrelated. Blank transition is the transition transferring assets _under_ different schema to a new UTXO set, which requires VM.
